src/firefly/js/tables/ui/TablePanel.css

```diff
@@ -161,19 +161,25 @@
     cursor: pointer;
 }
 
+.TablePanelOptionsWrapper {
+    min-width: 365px;
+    height: 450px;
+    min-height: 200px;
+    resize: both;
+    overflow: hidden;
+    position: relative;
+}
+
 .TablePanelOptions {
     position: absolute;
     padding: 5px 7px;
-    width: 350px;
-    top: 1px;
-    bottom: 1px;
-    right: 0;
-    background-color: beige;
-    border-left: 1px solid #CCC;
-    z-index: 100;
+    top: 5px;
+    bottom: 5px;
+    left: 5px;
+    right: 5px;
+    background-color: white;
```

Comment:
We might what to remove background-color setting here to match the look of FITS header popup.